Brick.ttl

```diff
@@ -3463,6 +3466,17 @@ brick:Min_Discharge_Air_Static_Pressure_Setpoint_Limit a owl:Class ;
         tag:Setpoint,
         tag:Static .
 
+brick:Min_Discharge_Air_Temperature_Setpoint a owl:Class ;
```

Comment:
Same as above. Why do we have this one without `_Limit`?

Reply:
Same as above